src/styles/designTokens.js

```diff
@@ -0,0 +1,29 @@
+// Colors
```

Comment:
Seems like some of below variables are unused in the project: `borderWidthThick` `spacingXLarge`